packages/wrangler/src/index.tsx

```diff
@@ -444,7 +444,9 @@ function createCLIParser(argv: string[]) {
           yesFlag ||
           (await confirm("Would you like to use git to manage this Worker?"));
         if (shouldInitGit) {
-          await execa("git", ["init"], { cwd: creationDirectory });
+          await execa("git", ["init", "--initial-branch=main"], {
```

Comment:
should this be `execaSync`?

Reply:
i don't think it matters too much, but sure if you like